src/gtfs/read.rs

```diff
@@ -859,56 +860,79 @@ where
+                                )),
+                            LogLevel::Warn
+                        );
+                        vec![stop_point]
+                    };
 
-                let from_stop_point = skip_error_and_log!(
-                    stop_points
-                        .get(&transfer.from_stop_id)
-                        .ok_or_else(|| format_err!(
-                            "Problem reading {:?}: from_stop_id={:?} not found",
-                            file,
-                            transfer.from_stop_id
-                        )),
-                    LogLevel::Warn
-                );
-
-                let to_stop_point = skip_error_and_log!(
-                    stop_points
-                        .get(&transfer.to_stop_id)
-                        .ok_or_else(|| format_err!(
-                            "Problem reading {:?}: to_stop_id={:?} not found",
-                            file,
-                            transfer.to_stop_id
-                        )),
-                    LogLevel::Warn
-                );
-
-                let (min_transfer_time, real_min_transfer_time) = match transfer.transfer_type {
-                    TransferType::Recommended => {
-                        let distance = from_stop_point.coord.distance_to(&to_stop_point.coord);
-                        let transfer_time = (distance / 0.785) as u32;
-
-                        (Some(transfer_time), Some(transfer_time + 2 * 60))
-                    }
-                    TransferType::Timed => (Some(0), Some(0)),
-                    TransferType::WithTransferTime => {
-                        if transfer.min_transfer_time.is_none() {
-                            warn!(
-                        "The min_transfer_time between from_stop_id {} and to_stop_id {} is empty",
-                        from_stop_point.id, to_stop_point.id
-                    );
-                        }
-                        (transfer.min_transfer_time, transfer.min_transfer_time)
+                let to_stop_points: Vec<&StopPoint> =
+                    if stop_areas.get(&transfer.to_stop_id).is_some() {
+                        stop_points
+                            .values()
+                            .filter(|stop_point| stop_point.stop_area_id == transfer.from_stop_id)
+                            .collect()
+                    } else {
+                        let stop_point = skip_error_and_log!(
+                            stop_points
+                                .get(&transfer.to_stop_id)
+                                .ok_or_else(|| format_err!(
+                                    "Problem reading {:?}: to_stop_id={:?} not found",
+                                    file,
+                                    transfer.to_stop_id
+                                )),
+                            LogLevel::Warn
+                        );
+                        vec![stop_point]
+                    };
```

Comment:
making the stop_point list (from_stop_points, to_stop_points) seems to be factorized

Reply:
Done in 66a8f8298aa9b0ac7b5767569e3608d6e79a11c7.